x-pack/plugins/maps/public/trigger_actions/visualize_geo_field_action.ts

```diff
@@ -76,7 +76,7 @@ const getMapsLink = async (context: VisualizeFieldContext) => {
         type: SOURCE_TYPES.ES_SEARCH,
         tooltipProperties: hasTooltips ? context.contextualFields : [],
         label: indexPattern.title,
```

Comment:
Does `indexPattern.title` still exist? Or should this be changed to get title from `context.dataViewSpec` as well?

Reply:
`indexPattern` be retrieved on L65 and should has actual title, the same as dataViewSpec.